crates/web5/src/dids/resolution/resolution_result.rs

```diff
@@ -31,6 +31,19 @@ impl ResolutionResult {
         match did.method.as_str() {
             "jwk" => DidJwk::resolve(uri),
             "dht" => DidDht::resolve(uri),
+            "web" => {
```

Comment:
why can't this be DidWeb::resolve ?

Reply:
or maybe make internal DidWeb::resolve do all this thread stuff

Reply:
yeah nice, agreed. done!